scripts/inference/run_mpt_with_ft.py

```diff
@@ -0,0 +1,476 @@
+# Copyright 2022 MosaicML LLM Foundry authors
```

Comment:
Naming question -- does this script support any FT model or only MPT? If the former I might suggest renaming this script to something like `ft_generate.py` to signal that this script is analogous to `hf_generate.py`

Reply:
This only supports MPT model.